src/features/stickyMessage/messageCooldown.ts

```diff
@@ -0,0 +1,79 @@
+import { Message } from 'discord.js'
+
+import { StickyMessage } from '@prisma/client'
+
+import { Environment } from '../../config.js'
+import { getCache, saveCache } from '../../utils/cache.js'
+
+import { STICKY_COOLDOWN_PREFIX, pushMessageToBottom } from './index.js'
+import { getCounter } from './messageCounter.js'
+
+interface IChannelCooldownContainer {
+  [channelId: string]: NodeJS.Timeout
+}
+
+const channelCooldown: IChannelCooldownContainer = {}
```

Comment:
I'd prefer using [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) instead of `{}`